Dockerfile

```diff
@@ -2,16 +2,17 @@
 
 FROM centos:centos7
 
-#
+WORKDIR /opt/arxiv
```

Comment:
Do we need this?

Reply:
Maybe not. By convention we use this is most of our downstream images, so thought it would be less redundant to put it here?

Reply:
While this is great to move shared ideas up in the chain, unless we can guarantee that in the future no one would add another `WRKDIR` towards the end of this image, we should avoid such things. If we do choose to keep it here, should we add a comment note above/below it saying, this image should have the final `WORKDIR` set so in order to avoid derived images to break silently.

Reply:
Gotcha. I think it makes sense to remove it, given that there are no obvious strong drivers to only and always use ``/opt/arxiv``.

Reply:
Fixed in 3701e20